pkg/forwarder/telemetry.go

```diff
@@ -76,23 +70,6 @@ func initEndpointExpvars() {
 	}
 }
 
-func initOrchestratorExpVars() {
-	for _, nodeType := range orchestrator.NodeTypes() {
-		transactionsIntakeOrchestrator[nodeType] = &expvar.Int{}
-		transaction.TransactionsExpvars.Set(nodeType.String(), transactionsIntakeOrchestrator[nodeType])
-	}
-	transaction.TransactionsExpvars.Set("OrchestratorManifest", &transactionsOrchestratorManifest)
-}
-
-func bumpOrchestratorPayload(nodeType int) {
-	e, ok := transactionsIntakeOrchestrator[model.K8SResource(nodeType)]
```

Comment:
Would it be possible to push `model.K8SResource(nodeType)` up the scope to avoid conditional compilation here? (I assume `model` is used there and that scope is already excluded from serverless).

If not, perhaps we should rather revert #15136 that introduce agent-payload use here, since it causes size regressions.